istioctl/cmd/webhook.go

```diff
@@ -0,0 +1,714 @@
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVar(&opts.enableValidationWebhook, "validation", true, "Enable "+
+		"validatation webhook (default true).")
+	flags.BoolVar(&opts.enableMutationWebhook, "injection", true, "Enable "+
+		"injection webhook (default true).")
+	// Specifies the local file path to webhook CA bundle.
+	// If empty, will read CA bundle from default Kubernetes service account.
+	flags.StringVar(&opts.caCertPath, "ca-bundle-file", "",
+		"PEM encoded CA bundle which will be used to validate the webhook's server certificates. "+
+			"If this is empty, the kube-apisever's root CA is used if it can be confirmed to have signed "+
+			"the webhook's certificates. This condition is sometimes true but is not guaranteed "+
+			"(see https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping)")
+	flags.StringVar(&opts.validationWebhookConfigPath, "validation-config-path", "",
+		"The file path of the validation webhook configuration.")
+	flags.StringVar(&opts.mutatingWebhookConfigPath, "injection-config-path", "",
+		"The file path of the injection webhook configuration.")
+	flags.StringVar(&opts.validatingWebhookConfigName, "validation-config", "istio-galley",
+		"The name of the ValidatingWebhookConfiguration to manage.")
+	flags.StringVar(&opts.mutatingWebhookConfigName, "injection-config", "istio-sidecar-injector",
+		"The name of the MutatingWebhookConfiguration to manage.")
+	flags.StringVar(&opts.validatingWebhookServiceName, "validation-service", "istio-galley",
+		"The service name of the validation webhook to manage.")
+	flags.StringVar(&opts.mutatingWebhookServiceName, "injection-service", "istio-sidecar-injector",
+		"The service name of the injection webhook to manage.")
+	flags.StringVar(&opts.webhookSecretName, "webhook-secret", "",
+		"The name of the secret of a webhook. istioctl will verify that the webhook certificate "+
+			"is issued by the CA certificate.")
+	flags.DurationVar(&opts.maxTimeForCheckingWebhookServer, "timeout", 60*time.Second,
+		"	Max time for checking the validating webhook server. If the validating webhook server is not ready"+
+			"in the given time, exit. Otherwise, apply the webhook configuration.")
+	flags.DurationVar(&opts.maxTimeForReadingWebhookCert, "read-cert-timeout", 60*time.Second,
+		"	Max time for waiting the webhook certificate to be readable.")
+
+	return cmd
+}
+
+func newDisableCmd() *cobra.Command {
+	opts := &disableCliOptions{}
+	cmd := &cobra.Command{
+		Use:   "disable",
+		Short: "Disable webhook configurations",
+		Example: `
+# Disable the webhook configuration of Galley
+istioctl experimental post-install webhook disable --validation --validation-config istio-galley
+# Disable the webhook configuration of Galley and Sidecar Injector
+istioctl experimental post-install webhook disable --validation --validation-config istio-galley 
+  --injection --injection-config istio-sidecar-injector
```

Comment:
Throughout, we should minimally show the common default case in the examples, e.g. 

```
# disable all webhooks
istioctl x post-install webhook disable

# disable all webhooks except injection
istioctl x post-install webhook disable --injection=false
```
The 2nd example has a double negative which might be a bit confusing. cc @esnible.

Reply:
Done.